third-party/chpl-venv/chpldoc-requirements1.txt

```diff
@@ -1,2 +1,2 @@
 # Split into 3 files to work around problems with CHPL_PIP_FROM_SOURCE
-MarkupSafe==2.1.3
+MarkupSafe==2.1.5
```

Comment:
We've typically noted such updates in our CHANGES file (either generally or individually), so it'd probably be good to either update your PR title to indicate you're modifying more than just Sphinx or list the version updates in your PR message